AsyncDisplayKit/ASTextNode.mm

```diff
@@ -470,6 +485,8 @@ - (id)_linkAttributeValueAtPoint:(CGPoint)point
    inAdditionalTruncationMessage:(out BOOL *)inAdditionalTruncationMessageOut
                  forHighlighting:(BOOL)highlighting
 {
+  ASDisplayNodeAssertMainThread();
```

Comment:
Yup, I think accessing the renderer's firstVisibleRange and grabbing _attributedText should be done in the lock? Separately, does it even make sense to have the _renderer method?